src/common/src/array/data_chunk.rs

```diff
@@ -56,6 +56,15 @@ impl From<usize> for Vis {
     }
 }
 
+impl From<&Vis> for Vis {
```

Comment:
This is just `clone`. With the suggestion above we do not even need to clone.

Reply:
Cool

Reply:
Can we remove this then?